src/commands/bot_status/bot_status.params.ts

```diff
@@ -0,0 +1,42 @@
+import { Choice, Param, ParamType } from '@discord-nestjs/core';
+
+export enum ActivityType {
+  Playing = 0,
+  Streaming = 1,
+  Listening = 2,
+  Watching = 3,
+  Custom = 4,
+  Competing = 5,
+}
+
+export enum ClientPresenceStatus {
+  Online = 'online',
+  Idle = 'idle',
+  DoNotDisturb = 'dnd',
```

Comment:
```suggestion
  DoNotDisturb = 'doNotDisturb',
```

Reply:
if we put value `doNotDisturb` it will not work, because we passing its value in discordjs to update status